lib/core/__tests__/anchors.test.js

```diff
@@ -109,3 +112,8 @@ test('Anchor index resets on each render', () => {
     'id="almost-unique-heading-1"'
   );
 });
+
+test('Anchor uses default renderer when empty', () => {
+  expect(render([{hLevel: 1}, {content: null}], 0, {}, {})).toEqual('<h1>');
```

Comment:
Why does it equal `<h1>` instead of `<h1></h1>`?

Reply:
Because the render method references the `rules.heading_open` alone `const render = md.renderer.rules.heading_open;`. We do not run the remarkable parser, simple invoke this one function. Am I right? The tests' expectations though checked to see if the opening header tag contained an `id` attribute, etc. So the new test's expectation looks unusual.